sdk/eventgrid/azure-resourcemanager-eventgrid/src/test/java/module-info.java

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+open module com.azure.resourcemanager.eventgrid {
```

Comment:
nice. this looks like something we should put into guidelines for test authoring.